src/Web/ViewModels/FraudProtectionIOModel.cs

```diff
@@ -12,6 +12,8 @@ public class FraudProtectionIOModel
 
         public string CorrelationId { get; set; }
 
+        public string EnvironmentId { get; set; }
```

Comment:
This doesn't appear to be used.

Reply:
yeah, may be added during investigating how to do this.